RELEASE_NOTES.rst

```diff
@@ -174,9 +174,10 @@ In this release, coala-bears has been revamped with new bears and more configs
 are added for existing bears.
 
```

Comment:
Wait, how is this not a bug fix, wasn't it a bug in the RELEASE_NOTES.rst file?
What else would this be called?